docs/developer/app-store/apps/emails-and-messages/sendgrid.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Sendgrid provider
+sidebar_position: 2
+---
+
+# Sendgrid provider
+
+:::info
+
+Using this provider requires a Sendgrid account. You can create one [here](https://signup.sendgrid.com/).
+
+:::
+
+Sendgrid is a cloud-based email delivery and management platform that provides businesses with a solution for sending transactional and marketing emails. It offers a set of tools and APIs that enable developers to integrate email functionality into their applications, websites, or software products.
+
+## Configuration
+
+Before you start using the App, configure your Sendgrid Account:
+
+- Generate [**API key**](https://docs.sendgrid.com/for-developers/sending-email/brite-verify#creating-a-new-api-key)
+
+- Create [**Sender**](https://docs.sendgrid.com/ui/sending-email/senders)
+
+- For every event message you want to send to the customers, create a [**Dynamic template**](https://docs.sendgrid.com/ui/sending-email/how-to-send-an-email-with-dynamic-templates#design-a-dynamic-template)
+
+:::tip
+
+Want to test messages? Working on the application code? See the [testing](#testing) section.
+
+:::
+
+Now you can configure EAM:
+
+1. Open the Emails and Messages app in Saleor Dashboard
+2. Click on `Add provider`
+3. Choose `Sendgrid`
+4. Provide configuration name and API key and click on `Save provider`
+5. Application will redirect automatically to the configuration details
+6. Navigate to _Sender_ section and choose sender. Save the changes
+7. In the events section, assign dynamic templates to events. Save the changes
+
+The application is now configured.
+
+## Testing
+
+Using a Gmail account for testing may not be possible due to security restrictions. To test the application, you can test the mailbox at [MailSlurp](https://www.mailslurp.com/).
+
+1. Create an account at [MailSlurp](https://www.mailslurp.com/)
+2. Create a new inbox - [documentation](https://www.mailslurp.com/guides/creating-inboxes/)
+3. Use the inbox address to create a new sender in Sendgrid - [documentation](https://docs.sendgrid.com/ui/sending-email/senders)
```

Comment:
**polish**: I think it looks nicer to link to the docs right from the corresponding fragment. 

```suggestion
3. Use the inbox address to [create a new sender in Sendgrid](https://docs.sendgrid.com/ui/sending-email/senders)
```